spec/views/user/index.html.erb_spec.rb

```diff
@@ -4,11 +4,11 @@
   describe 'Index page' do
     before(:each) do
       @user1 = User.create(name: 'Gul',
-                           photo: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRJHYna2rR4LtBNlA3gUSGxsmaiYY4PyxQEcA&usqp=CAU', bio: 'Spring Sun', posts_counter: 0)
+                           photo: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRJHYna2rR4LtBNlA3gUSGxsmaiYY4PyxQEcA&usqp=CAU', bio: 'Spring Sun', email: 'example1@example', posts_counter: 0)
       @user2 = User.create(name: 'Ahmad',
-                           photo: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRJHYna2rR4LtBNlA3gUSGxsmaiYY4PyxQEcA&usqp=CAU', bio: 'Spring Sun', posts_counter: 0)
+                           photo: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRJHYna2rR4LtBNlA3gUSGxsmaiYY4PyxQEcA&usqp=CAU', bio: 'Spring Sun', email: 'example2@example', posts_counter: 0)
       @user3 = User.create(name: 'Muzafar',
-                           photo: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRJHYna2rR4LtBNlA3gUSGxsmaiYY4PyxQEcA&usqp=CAU', bio: 'Spring Sun', posts_counter: 0)
+                           photo: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRJHYna2rR4LtBNlA3gUSGxsmaiYY4PyxQEcA&usqp=CAU', bio: 'Spring Sun', email: 'example3@example', posts_counter: 0)
```

Comment:
- [ ] The test in this file is failing.

  ![ffis](https://user-images.githubusercontent.com/36542986/228454842-50579b56-474e-4671-94d4-0b11b9365874.jpg)

  If you look at your User model in the console there is no `email` attribute for the user but you are trying to create a user with this attribute.

  ![mduw](https://user-images.githubusercontent.com/36542986/228455232-8d6abc6b-6eb6-4405-a279-8f2a61332614.jpg)

  You can fix this by removing `email` attributes from these lines.